clustering/clustering_example.Rmd

```diff
@@ -40,8 +40,8 @@ Create output folder.
 
 ```{r}
 # Create the plots folder if it doesn't exist
-if (!dir.exists("plots")) {
-  dir.create("plots")
+if (!dir.exists("plots")) { # Replace with path to desired output plots directory
```

Comment:
For these kinds of things we might want declare the file paths once and assign them, then supply the assignment. I could see people forgetting to change this in both places. We probably should have done this the first time around.